frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsDescriptionText.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text } from '@chakra-ui/react'
+
+import { GUIDE_SPCP_ESRVCID } from '~constants/links'
+import Link from '~components/Link'
+
+export const AuthSettingsDescriptionText = () => {
+  return (
+    <Text
+      textStyle="subhead-1"
+      color="secondary.500"
+      marginBottom="40px"
+      marginTop="40px"
```

Comment:
Avoid using `px`. 

`40px` is equivalent to `2.5rem` for this repo. We can also use `my` as a shorthand to represent `marginBottom` + `marginTop` 😄 

```suggestion
      my="2.5rem"
```

Reply:
resolved